packages/react-core/src/components/ProgressStepper/examples/ProgressStepperBasicWithDescription.tsx

```diff
@@ -15,7 +15,13 @@ export const ProgressStepperBasicWithDescription: React.FunctionComponent = () =
     <ProgressStep
       variant="info"
       isCurrent
-      description="This is the second thing to happen"
+      description={
+        <>
+          This is the second thing to happen
+          <br />
+          In progress
+        </>
+      }
```

Comment:
I'm thinking this could be tweaked slightly. Maybe either updating that first line to "This is the second thing to happen with a line break", or swapping the two lines (so "In progress" > line break > "This is the second thing to happen") to more closely resemble the image shared in the original issue.

Mainly I just wonder whether, visually, the example will appear as a normal line break rather than forcing the line break with `<br />`. What do you think @andyyvo ?